### PR TITLE
DOCSP-22962 New Explain Plan for Aggregation Pipeline Builder

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
You can now view an explain plan on Compass while you are building your aggregation pipelines.

JIRA: https://jira.mongodb.org/browse/DOCSP-22962

STAGING: https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-22962/aggregation-pipeline-builder/#explain-plan-for-pipelines

### CODE REVIEW

File: source/aggregation-pipeline-builder.txt [hunk 1]
```diff
@@ -159,6 +159,25 @@ To view and change pipeline settings:
 
 .. _pipeline-custom-collation:
 
+Explain Plan for Pipelines
+--------------------------
+
+Use the :guilabel:`Explain Plan` to help you understand the performance of 
+your pipelines. You can view your :guilabel:`Explain Plan` at any point while
```
Comment (jason-price-mongodb) [inline source/aggregation-pipeline-builder.txt:166]: Perhaps "You can view the :guilabel:`Explain Plan` when 
creating or editing your pipeline".

File: source/aggregation-pipeline-builder.txt [hunk 2]
```diff
@@ -159,6 +159,25 @@ To view and change pipeline settings:
...
+your pipelines. You can view your :guilabel:`Explain Plan` at any point while 
+creating or editing your pipeline. 
+
+To view your pipeline :guilabel:`Explain Plan`, click :guilabel:`Explain` in the 
+right corner of your pipeline toolbar.
+
+.. figure:: /images/compass/agg-explain-plan.png
+    :alt: Aggregation Pipeline Builder Explain Plan
+
+Your results are presented in the following format:
```
Comment (jason-price-mongodb) [inline source/aggregation-pipeline-builder.txt:175]: "in this format"

File: source/aggregation-pipeline-builder.txt [hunk 3]
```diff
@@ -159,6 +159,25 @@ To view and change pipeline settings:
 
 .. _pipeline-custom-collation:
 
+Explain Plan for Pipelines
+--------------------------
+
+Use the :guilabel:`Explain Plan` to help you understand the performance of
```
Comment (jason-price-mongodb) [inline source/aggregation-pipeline-builder.txt:165]: Perhaps "Use the :guilabel:`Explain Plan` panel ..."?
Not sure what the convention is for naming in the GUI. Perhaps examine how other similar items are named?

**Review by jason-price-mongodb — APPROVED** [review]
@jocelyn-mendez LGTM with suggestions and question about naming standards. Cheers.

File: source/aggregation-pipeline-builder.txt (hunk 3 above)
Comment (jocelyn-mendez1) [reply source/aggregation-pipeline-builder.txt]: Done.

File: source/aggregation-pipeline-builder.txt (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/aggregation-pipeline-builder.txt]: Done.

File: source/aggregation-pipeline-builder.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/aggregation-pipeline-builder.txt]: Done.

**Review by mabaasit — APPROVED** [review]

**Pull request merged**